### PR TITLE
Check if async is available before loading async-to-gen

### PR DESCRIPTION
**Pull request by timneutkens**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,9 +276,8 @@ function that returns a URL with an ephemeral port every time it's called.
 
 ### Transpilation
 
-We now use [async-to-gen](https://github.com/leebyron/async-to-gen),
-so that the only transformation that happens is converting `async`
-and `await` to generators.
+We now use [is-async-supported](https://github.com/timneutkens/is-async-supported) combined with [async-to-gen](https://github.com/leebyron/async-to-gen),
```
Comment (albinekb) [inline README.md]: neat stuff 🎉

File: bin/micro [hunk 2]
```diff
@@ -57,7 +58,9 @@ if ('/' !== file[0]) {
   file = resolve(process.cwd(), file)
 }
 
-require('async-to-gen/register')
+if(!isAsyncSupported()) {
```
Comment (albinekb) [inline bin/micro]: Add a space for consistency (looks like there are spaces further down in this file)
```js
if (!isAsyncSupported()) {
```

**Review by albinekb — APPROVED** [review]

File: bin/micro (hunk 2 above)
Comment (timneutkens) [reply bin/micro]: Added 🔥

File: README.md (hunk 1 above)
Comment (rauchg) [inline README.md]: The temporal reference ("now") is only interesting to people who already knew about the implementation details. I'd take it out

Comment (timneutkens) [reply README.md]: Done 👍

**Comment by rauchg** [comment]
Great work @timneutkens

**Pull request merged**